cohortextractor/concepts/tables.py

```diff
@@ -7,6 +7,3 @@ class ClinicalEvents(QueryBuilder):
 
     def __init__(table_name):
         super().__init__("clinical_events")
-
-
-clinical_events = ClinicalEvents()
```

Comment:
I think maybe this was one of those places I mentioned earlier today, where I was trying too hard to implement exactly what was written int he code snippets in the original ticket.  I thought perhaps we were trying not to make users have to instantiate the table, but I realise we were already making them instantiate a `Cohort()` so it's probably better to be explicit about it anyway

Reply:
I think there's a semantic difference. Their purpose is to create a cohort, so it seems reasonable for them to construct a cohort object. But they're not creating the tables, they're accessing tables that already exist.

Reply:
I think I prefer the original from a usability perspective.  I think it's quite likely that users will be confused about what things are classes and what things are instances, and the fewer things they have to instantiate, the better.